docs/pricing/quotas/manage-event-stream-guide.mdx

```diff
@@ -242,7 +242,7 @@ To learn more and see code samples, check out:
 
 This integration attaches global handlers to capture uncaught exceptions (`onerror`) and unhandled rejections (`onunhandledrejection`). Both handlers are enabled by default, but can be disabled through configuration. Learn more in [GlobalHandlers Integration](/platforms/javascript/configuration/integrations/globalhandlers/).
 
-Check out additional configuration options with the [tryCatch](/platforms/javascript/configuration/integrations/trycatch/) and [ReportingObserver](/platforms/javascript/configuration/integrations/reportingobserver/) integrations.
+Check out additional configuration options with the [tryCatch](/platforms/javascript/configuration/integrations/browserapierrors/) and [ReportingObserver](/platforms/javascript/configuration/integrations/reportingobserver/) integrations.
```

Comment:
We should also update the link text to `browserApiErrors` (currently still `tryCatch`)

Reply:
also a little nit pick: casing is different `tryCatch` vs `ReportingObserver`